paper.tex

```diff
@@ -217,20 +251,46 @@ \section*{ARG inference methods}
 to infer ARGs. We focus our attention on some recent methods and discuss the
 properties of the ARGs inferred.
 
-\begin{itemize}
-\item Early methods \citep{hein1990reconstructing}
-\item Those sampling from the coalescent with recombination
-\citep{griffiths1996ancestral,kuhner2000maximum,fearnhead2001estimating},
-or SMC
-\citep{rasmussen2014genome}
-\item Hueristic methods~\citep{song2005efficient,minichiello2006mapping,
-parida2008estimating,fallon2013acg,mirzaei2016rent}
-\item Discussions of the complexity of inferring ARGs (but we need to
-dig into this: is this the complexity of sampling from the coalescent,
-finding a ``min ARG'', or something else?
-\citep{bordewich2005computational,wang2001perfect,hein1990reconstructing,song2005constructing}
-\item Recent methods: Relate, tsinfer, tsdate, etc.
-\end{itemize}
+The ARG has been used as a model for recombining DNA sequence data essentially
+since it was first formulated. Early methods focused on finding parsimonious
+ARGs consistent with a given set of
+sequences \citep{hein1990reconstructing, lyngso2005minimum}.
+However, identifying the most parsimonious ARG for a given sequence data set
+is NP-hard \citep{wang2001perfect}, and subsequent methods have resorted to
+various heuristics \citep{fallon2013acg, hein1993heuristic, ignatieva2021kwarg, 
+kelleher2019inferring, minichiello2006mapping, mirzaei2017rent,
+parida2008estimating, song2005efficient, thao2019hybrid, zhang2021biobank}. 
+The scalability of these methods varies substantially based on the heuristic in use
+and the underlying data structure, but the most efficient methods can be practically
+used on megabase scale data and tens of thousands of samples under human-like
+parameters \citep{kelleher2019inferring, zhang2021biobank}.
+
+An alternative approach is to treat the ancestry as a latent parameter to be averaged out
+by Monte Carlo methods, based either on importance sampling 
+\citep{griffiths1996ancestral, fearnhead2001estimating, jenkins2011inference} 
+or MCMC \citep{kuhner2000maximum, nielsen2000estimation, wang2008bayesian}.
+These methods invariably operated on a representation of the ``little ARG", typically
+generated until the grand MRCA. They are extremely computationally expensive,
```

Comment:
Really? As in, the first common ancestor of all nodes in the Big ARG?

Reply:
Ah, no. I might be misusing GMRCA. I mean the first node which is a common ancestor at all sites, without pruning away sites as soon as their local MRCA is reached (see Figure 1 [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1461855/pdf/11729171.pdf)). I think I should push one more commit with a slightly longer description to that effect, and some partial tweaks based on the comments above.

Reply:
That's my understanding of the GMRCA - there's nothing grand about the last node that happens to be an MRCA in the little ARG.